packages/api/utils/database.ts

```diff
@@ -80,7 +85,7 @@ export async function createConnection(): Promise<Connection> {
     },
     logging: "all",
     database: databaseName,
-    entities: ["/opt/nodejs/orm/entity/**.js"],
+    entities: [Portfolio, Application, Environment, TaskOrder, Clin, "/opt/nodejs/orm/entity/**.js"],
     migrations: ["/opt/nodejs/orm/migration/**.js"],
```

Comment:
Wait a second... has this seriously been an option the whole time? To just put the types directly in this array?

Reply:
Yep.  See the examples for _entities_ at [Common connection options](https://typeorm.io/#/connection-options/common-connection-options).  I struggled with EntityMetadataNotFoundError until I did this.